### PR TITLE
Added react-foundation-apps and react-slick

### PR DESCRIPTION
**Pull request by akiran**

### CODE REVIEW

File: files/react-foundation-apps/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "react-foundation-apps",
+  "repo": "akiran/react-foundation-apps",
+  "files": {
+    "include": ["./dist/*.js"]
```
Comment (jimaek) [inline files/react-foundation-apps/update.json]: Use `basePath` to set dist/ and then in include just use `*.min.js`

File: files/react-slick/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "react-slick",
+  "repo": "akiran/react-slick",
+  "files": {
+    "include": ["./dist/*.js"]
```
Comment (jimaek) [inline files/react-slick/update.json]: Same thing: Use `basePath` to set dist/ and then in include just use `*.min.js`

**Comment by jimaek** [comment]
Please rename the directories. react.slick and react.foundation-apps

**Comment by akiran** [comment]
@jimaek Made suggested changes

**Comment by jimaek** [comment]
Great, now please squash the commits into 1 and its good to go

**Comment by jimaek** [comment]
Actually no, your basPath has wrong format. Check out here the correct https://github.com/jsdelivr/libgrabber#examples

After you fix it squash the commits

**Comment by akiran** [comment]
Couldn't squash it because of upstream merge.
I will submit a new pull request

**Pull request closed without merging**